azure-pipelines-pr-validation.yml

```diff
@@ -264,6 +264,31 @@ extends:
             arguments: '-configuration $(BuildConfiguration) -branchName "$(SourceBranchName)" -prValidation'
           condition: succeeded()
 
+        # Microbuild uses the build number in the drop name, and it doesn't like the fancy build number.
+        # Rest it here before we publish, then set it back when we're done.
+        - powershell: Write-Host "##vso[build.updatebuildnumber]$(OriginalBuildNumber)"
+          displayName: Reset BuildNumber
+          condition: succeeded()
+
+        - powershell: ls $(Build.SourcesDirectory)\artifacts\tmp\$(BuildConfiguration)\SymbolPackages
+          displayName: show symbols
+
+        # Use microbuild to publish symbols only to the internal feeds for PR val builds.
+        # We do not use arcade/darc as by default they get published to public feeds, and it is complicated
+        # to configure PR validation builds to push to an internal only channel.
+        - task: MicroBuildArchiveSymbols@5
+          displayName: 'Upload Symbols to Symweb/MSDL'
+          inputs:
+            SymbolsFeatureName: 'RoslynPRValidation'
+            SymbolsProject: 'DDE'
+            SymbolsAgentPath: '$(Build.SourcesDirectory)\artifacts\tmp\$(BuildConfiguration)\SymbolPackages'
+            SubmitToInternet: false
+          condition: succeeded()
+
+        - powershell: Write-Host "##vso[build.updatebuildnumber]$(BuildNumber)"
```

Comment:
```suggestion
        - powershell: Write-Host "##vso[build.updatebuildnumber]$(FancyBuildNumber)"
```

Reply:
FancyBuildNumber doesnt actually exist here.  FancyBuild step seems to set BuildNumber instead

Reply:
Sorry, saw it in the remove code below. Double checked and you are absolutely right.